liquibase-core/src/main/java/liquibase/executor/jvm/ColumnMapRowMapper.java

```diff
@@ -24,6 +24,12 @@
 @SuppressWarnings({"unchecked"})
 public class ColumnMapRowMapper implements RowMapper {
 
+    private final boolean caseSensitiveDatabase;
```

Comment:
This caseSensitive change looks unrelated to this PR #2190 ?
Probably no harm to include it if you were planning to add this anyway, ofc

Reply:
Thanks for noticing that. Yes, I hadn't cleaned my master branch well enough before updating yours. 

That PR should go in first, so I won't bother fixing the git history and force pushing. So it should be fine as it is.